meson.build

```diff
@@ -107,3 +112,16 @@ if scdoc.found()
 		)
 	endforeach
 endif
+
+if get_option('zsh-completions')
+	zsh_files = files(
+		'completions/zsh/_makoctl',
+                'completions/zsh/_mako'
```

Comment:
Mixed indent here.

Reply:
I've should have fixed this issue in 05758b6298f6c69028fbb961b44d7a01ccdb8fd4 by using editorconfig, thx.